examples/README.md

```diff
@@ -0,0 +1,8 @@
+<p align="center">
+    <img src="../etc/images/Primary_logo_blue.png" height="180">
+</p>
+
+# Helidon Examples
+
+Helidon 4 examples have moved to the [helidon-examples](https://github.com/helidon-io/helidon-examples/tree/helidon-4.x) repository.
```

Comment:
Another candidate for using the hypothetical new attribute for the last part of the path.